torch/nn/quantized/modules/linear.py

```diff
@@ -197,8 +191,9 @@ def __setstate__(self, state):
     def weight(self):
         return torch.ops.quantized.linear_unpack(self._packed_weight)
 
-    def set_weight(self, w):
-        self._packed_weight = torch.ops.quantized.linear_prepack(w)
+    def set_weight_bias(self, w, b):
```

Comment:
if you still want to keep self.bias for convenience (though we shouldn't really) - let's put this assignment here. But I'm bothered by this discrepancy between bias and weight in unpacking

Reply:
Have a similar concern. Lets eliminate self.bias as a member and only have packed_params.
Instead have two functions: 
```python
def weight(self):
        (w,b) =  torch.ops.quantized.linear_unpack(self._packed_params)
        return w

def bias(self):
        (w,b) =  torch.ops.quantized.linear_unpack(self._packed_params)
        return b
```